c++/src/SchemaEvolution.hh

```diff
@@ -0,0 +1,45 @@
+#ifndef ORC_SCHEMA_EVOLUTION_HH
+#define ORC_SCHEMA_EVOLUTION_HH
+
+#include <unordered_map>
+#include <unordered_set>
+#include "orc/Type.hh"
```

Comment:
It is recommended that this line ' #include "orc/Type.hh" ' be moved to precede line ' #include <unordered_map>'.

Reply:
done